tox.ini

```diff
@@ -14,26 +14,17 @@ passenv =
     JAVA_HOME
 
 deps =
-    coverage>=4.2b1
-    flake8
-    flake8_docstrings
-    mock
-    mar
-    scriptworker
-    pytest
-    pytest-asyncio>=0.6.0
-    pytest-cov
-    pytest-mock
+    -r requirements/test.txt
 
 commands=
     py.test --cov-config .coveragerc --cov=signingscript --cov-report term-missing
     coverage html
     flake8
+    pip-compile-multi verify
```

Comment:
We may not want to do this unless we restrict ourselves to a single version of python and a single host arch. (The generated pins and hashes can differ between python 3.6 and 3.7, or between linux and macosx). We should probably have a separate target for this test if we want to run it.

Instead, we should probably test against the unpinned deps, and have a separate process or test for pinning.

Reply:
Hmm, I thought that the list of hashes contains all possible combinations (tarball, version/platform specific wheels) for that particular version...

Reply:
So I think if we run this on all supported python versions (just as it is now in the patch), we should be safe. I got red tox runs until I fixed the issue with my initial compilation that I made on py37, but it was not sufficient for py36.

Reply:
I may be misunderstanding, but aiui:

- we land pinned hashes and versions
- these hashes and versions will be tied to a python minor version and OS arch
- if we run this test on a different python minor version and OS arch, we may hit issues